packages/generator-helper/src/__tests__/generatorHandler.test.ts

```diff
@@ -61,7 +61,9 @@ function getExecutable(name: string): string {
 }
 
 describe('generatorHandler', () => {
-  // TODO: Windows: this test fails with timeout.
+  // TODO: Windows: this test fails with ENOENT on CI because it can't file the .cmd file:
+  //   spawn D:\\a\\prisma\\prisma\\packages\\generator-helper\\src\\__tests__\\exiting-executable.cmd ENOENT
+  // This does not happen on Windows locally.
```

Comment:
The timeout was fixed, now this test has the same problem as the next one.

On CI Windows seemingly can't find `exiting-executable.cmd` and `invalid-executable.cmd` in this folder, but these two tests pass for me locally on Windows if I un-skip them, so I couldn't investigate.

The `.cmd` files look identical to the ones in the tests that pass on CI as well, and the paths aren't too long (in fact, they are even longer in some tests that pass) so no idea what could go wrong.